CodingPractice/PythonAssignments/cleancode/BankAccount.py

```diff
@@ -3,26 +3,23 @@
 
 class BankAccount:
 
-    def __init__(self, acct_num, balance):
+    def __init__(self, acct_num, acct_balance):
         self.acct_num = acct_num
-        self.balance = balance
+        self.balance = acct_balance
 
     pence = Decimal('.01')
 
-    def __add__(self, other):
-        account_balance = Decimal(self.balance)
-        other_balance = Decimal(other.balance)
-        new_balance = account_balance + other_balance
-        return new_balance.quantize(self.pence, rounding=ROUND_HALF_UP)
+    def __add__(self, other_acct):
+        acct_balance, other_acct_balance = Decimal(self.balance), Decimal(other_acct.balance)
+        new_acct_balance = acct_balance + other_acct_balance
+        return new_acct_balance.quantize(self.pence, rounding=ROUND_HALF_UP)
 
-    def __sub__(self, other):
-        account_balance = Decimal(self.balance)
-        other_balance = Decimal(other.balance)
-        new_balance = account_balance - other_balance
+    def __sub__(self, other_acct):
+        acct_balance, other_acct_balance = Decimal(self.balance), Decimal(other_acct.balance)
```

Comment:
Converting to Decimal repeats where could you put this so it only happens once per instance instead of in each function ?